gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -464,28 +464,34 @@ MM_MemorySubSpaceSemiSpace::tearDown(MM_EnvironmentBase *env)
 void
 MM_MemorySubSpaceSemiSpace::flip(MM_EnvironmentBase *env, Flip_step step)
 {
+	Trc_MM_MSSSS_flip_entry(env->getLanguageVMThread());
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
 	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
 	bool debug = _extensions->debugTiltedScavenge;
```

Comment:
remove 'debug' local
we should also remove debugTiltedScavenge from Extensions, but I know that OpenJ9 project is using it, so that should be removed first (before this PR can proceed) in a separate J9 PR

Reply:
actually we can't remove debugTiltedScavenge from Extensions yet, since it's used for a few other prints (in MM_PhysicalSubArenaVirtualMemorySemiSpace). perhpas we can convert those prints too, but let's finish this first

Reply:
We can remove the debug flag just locally since the unused variable warning will become an error here.